### PR TITLE
Update docs with actual MariaDB JSON support

### PR DESCRIPTION
**Pull request by DefteZ**

### CODE REVIEW

File: lib/sqlalchemy/dialects/mysql/json.py [hunk 1]
```diff
@@ -13,8 +13,7 @@
 class JSON(sqltypes.JSON):
     """MySQL JSON type.
 
-    MySQL supports JSON as of version 5.7.  Note that MariaDB does **not**
-    support JSON at the time of this writing.
+    MySQL supports JSON as of version 5.7.  MariaDB support JSON (as alias for LONGTEXT) from version 10.2
```
Comment (CaselIT) [inline lib/sqlalchemy/dialects/mysql/json.py:16]: We have a max line length of 80char. Could you please reformat this line?

**Review by CaselIT — CHANGES_REQUESTED** [review]

**Comment by CaselIT** [comment]
Thanks for the correction.

File: lib/sqlalchemy/dialects/mysql/json.py (hunk 1 above)
Comment (zzzeek) [reply lib/sqlalchemy/dialects/mysql/json.py:16]: needs grammatical correction as well.

Comment (DefteZ) [reply lib/sqlalchemy/dialects/mysql/json.py:16]: @zzzeek could you please suggest your variant?

Comment (DefteZ) [reply lib/sqlalchemy/dialects/mysql/json.py]: @CaselIT done

Comment (CaselIT) [reply lib/sqlalchemy/dialects/mysql/json.py]: I think he was talking about `support`->`supports` like in the MySQL version

Comment (zzzeek) [reply lib/sqlalchemy/dialects/mysql/json.py]: "MariaDB supports JSON (as an alias for LONGTEXT) as of version 10.2."

Comment (DefteZ) [reply lib/sqlalchemy/dialects/mysql/json.py]: @zzzeek , my bad :)

All are fixed. Thanks

**Review by CaselIT — APPROVED** [review]
looks good to me now

**Comment by zzzeek** [comment]
merged directly thanks!

**Pull request merged**